presto-main/src/main/java/com/facebook/presto/sql/planner/GroupingOperationRewriter.java

```diff
@@ -50,14 +47,14 @@
     private final QuerySpecification queryNode;
     private final Analysis analysis;
     private final Metadata metadata;
-    private final Map<QuerySpecification, Symbol> groupIdSymbols;
+    private final Optional<Symbol> groupIdSymbol;
```

Comment:
Yes, in the case of a simple `GROUP BY`, the `grouping()` function can still be used but there will be no `groupId` symbol